assignments/arrays.js

```diff
@@ -63,34 +63,65 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
 
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+
 let lastCar = 0;
-console.log();
+
+for (i = 0; i < lastCar.length; i++) {
+  lastCar[i]; 
+}
+
+console.log(`The make is ${inventory[i].car_make} and the model is ${inventory[i].car_model}`);
```

Comment:
when I run it I'm getting an answer of "The make is Lincoln and the model is Navigator" but on line 54 of the array it looks like the car_model should be "Town Car"

look into this because you're not getting the right answer. (you're getting the first item in the array, not the last).